library_conversion/tools/library_conversion/library_conversion.py

```diff
@@ -3,15 +3,41 @@
 import os
 import sys
 import argparse
-import nist_msp_conversion as msp_convert
-import mzvault_conversion as mzvault_convert
+from enum import Enum
+
+
+# Specifies the input formats
+class InputFormat(Enum):
+    mzvault, nist_msp, bmdms = range(3)
+
+# handle formats and convert
+def convert(libformat, input_filename, mgf_filename, batch_filename, pi_name, collector_name):
```

Comment:
We do not need these other formats. This is only meant for mzvault.

Reply:
I added a test file for other msp formats that are compatible with the NIST specs.
This format is also used by MZmine and other tools to create libraries.